share/functions/fish_default_key_bindings.fish

```diff
@@ -1,6 +1,10 @@
 
 function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fish" -a mode
 	if not set -q mode[1]
```

Comment:
While you're touching this remove the `-a mode` and change this to `if not set -q argv[1]`. The mode arg isn't even the mode, it's the `-M` switch when args are passed to the function, and the var isn't used anywhere else.